zenodo/modules/pages/utils.py

```diff
@@ -81,9 +85,14 @@ def send_support_email(context):
         body=msg_body
     )
 
-    if context['form'].attachments.data:
-        msg.attach(context['form'].attachments.data.filename,
-                   'application/octet-stream',
-                   context['form'].attachments.data.read())
+    attachments = request.files.getlist("attachments")
+    if attachments:
+        for upload in attachments:
+            if len(upload.read()) > \
```

Comment:
This reads the whole file first (which might be eg. 5GB) and then rejects. There are other ways to prevent this from happening beforehand (@jainaman224 don't worry with resolving this for the time being, this is just a note)